thunder/dynamo/benchmark_utils.py

```diff
@@ -139,7 +139,8 @@ def __init__(self, specification_name="nvfuser"):
     # Returns the nvFuser callable from the nvFuser bound symbol.
     # See the TODO in :class:`thunder.dynamo.report.ThunderFusionReport` for more details.
     def compile(self, nvfusion_bsym):
-        return nvfusion_bsym._call_ctx[nvfusion_bsym.sym.name]
+        fd = nvfusion_bsym._call_ctx[nvfusion_bsym.sym.name].last_used
+        return fd.execute
```

Comment:
This can probably be changed to 
```python
return lambda *args: fd.execute(args)
```

With this change, `if compile_fn.name == "nvfuser"` in `report.py` shouldn't be required.

Though a question would be that this will add a slight overhead of calling into the lambda first, not sure if that is acceptable.